calico/operations/fips.mdx

```diff
@@ -0,0 +1,62 @@
+---
+description: Run Calico using FIPS validated cryptography.
+---
+
+# FIPS mode
+
+## Big picture
+
+Run {{prodname}} in [FIPS 140-2](https://csrc.nist.gov/publications/detail/fips/140/2/final) compliant mode.
+
+## Value
+
+When running in FIPS compliant mode, {{prodname}} uses FIPS-approved cryptographic algorithms and NIST-validated cryptographic modules.
+
+## Concepts
+
+The Federal Information Processing Standards (FIPS) are publicly announced standards developed by the National Institute of Standards and Technology for use in computer systems by government agencies and government contractors. {{prodname}} FIPS mode is enabled during installation by:
+
+- Switching the cryptographic modules for the golang-based applications to use the FIPS-140-2 validated [Tigera Cryptographic Module](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4095)
+- Configuring TLS servers and other cryptographic functions to use FIPS 140-2 approved cryptographic algorithms
+- Leveraging third-party cryptographic modules for other applications:
```

Comment:
You can remove line 20+21, those are not used in OSS.

Reply:
Also fluentd not used in OSS

Reply:
I meant remove:
```
- Leveraging third-party cryptographic modules for other applications:
- Fluentd is built using [Red Hat Enterprise Linux 8 OpenSSL Cryptographic Module](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/3842)
```

Reply:
Ah, right! Removed.